src/containers/ClusterInfo/ClusterInfo.tsx

```diff
@@ -128,39 +156,56 @@ export const ClusterInfo = ({clusterTitle, additionalClusterInfo = []}: ClusterI
         return info;
     };
 
-    if (loading && !wasLoaded) {
+    if ((loading && !wasLoaded) || (nodesLoading && !nodesWasLoaded)) {
```

Comment:
I didn't merge `clusterNodes` into `cluster` state, as we plan to change the navigation and `Versions` will be a separate tab. `clusterNodes` will be requested in `Cluster`, loading state will be different for `ClusterInfo` and `Versions`